src/sass/phone-input/phone-input.scss

```diff
@@ -10,7 +10,14 @@
     border-radius: 8px 0 0 8px;
     min-width: 64px;
     padding-inline: var(--spacing-200) var(--spacing-100);
-    border-right: none;
+    border-right-style: revert;
+}
+
+.phone-input button.btn:focus-visible {
+    box-shadow: 0 0 0 0.5pt var(--color-stroke-strong);
```

Comment:
`pt` is actually better suited for print. Can we use `px` instead?

Reply:
I didn't choose `pt`, it's what we had before. I can change to `px` though, seems more sensible.